assets/app/styles/_components.less

```diff
@@ -202,40 +201,69 @@
       .osc-object-highlight();
       .flex(@columns: 1 1 auto);
       max-width: 450px;
+      min-width: 90px;
       border-radius: @border-radius-md;
-      border: 1px solid transparent;
-      // box-shadow: @osc-object-box-shadow; <- remove if it's not used in highlight
+      border: 1px solid #fff;
```

Comment:
I set it to white because when set to transparent the repeating-linear-gradient edges shows through in offset manner, making it looks strange. e.g. 
![screen shot 2015-06-01 at 11 01 48 am](https://cloud.githubusercontent.com/assets/1874151/7915881/6a7a67da-084e-11e5-8ec4-1a6920fff1bb.png)

Currently the pods page shows pod details on the tile bg. 

I'll look for another option.